Makefile

```diff
@@ -54,7 +54,11 @@ TANZU_FRAMEWORK_REPO_BRANCH ?= v0.10.0
 TANZU_FRAMEWORK_REPO_HASH ?=
 # TKG_DEFAULT_IMAGE_REPOSITORY override for using a different image repo
 ifndef TKG_DEFAULT_IMAGE_REPOSITORY
-TKG_DEFAULT_IMAGE_REPOSITORY ?= projects-stg.registry.vmware.com/tkg
+TKG_DEFAULT_IMAGE_REPOSITORY ?= projects.registry.vmware.com/tce
+endif
+# TKG_DEFAULT_COMPATIBILITY_IMAGE_PATH override for using a different image path
+ifndef TKG_DEFAULT_COMPATIBILITY_IMAGE_PATH
+TKG_DEFAULT_COMPATIBILITY_IMAGE_PATH ?= compatability/v0.10.0-1/tkg-compatibility
```

Comment:
this is being thrown in the mispell checker:
```
Makefile:61:40: "compatability" is a misspelling of "compatibility"
```
do we care? or do we wanna push a new image?

Reply:
i'm pushing a new image now - please hold 👀

Reply:
Done! Pushed a new one at
```
compatibility/v0.10.0/tkg-compatibility
```